lib/index.js

```diff
@@ -61,69 +64,57 @@ exports = module.exports = internals.Connection = class {
     validateSegmentName(name) {
 
         if (!name) {
-            throw new Boom.Boom('Empty string');
+            throw new Error('Empty string');
```

Comment:
If we write a changelog for this version, we should probably document this breaking change as well

Reply:
As a start, I've added this change in the PR description 👍

Reply:
Alright, thanks.